### ISSUE TITLE
Adding new plugins from computations

### ISSUE THREAD

**Issue by JosePizarro3** [FAIRmat-NFDI/.github#5]
@Pepe-Marquez can I modify the README to add the new plugins?

**Comment by Pepe-Marquez** [comment]
> @Pepe-Marquez can I modify the README to add the new plugins?

Sure! Put a PR and make sure that the comp part is not so overwhelming that it screens out the experimental ones, please 🙏!

Feel free also to add something about the plugin template repo.

**Comment by JosePizarro3** [comment]
Yeah thanks, so I need to be added as maintainer or collaborator to push something.
